@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/core/fn_coreGetWCAs.sqf

```diff
@@ -263,5 +263,8 @@ if (_heli getVariable "fza_ah64_rtrbrake") then {
 if (_fcrDamage >= SYS_FCR_DMG_THRESH) then {
     _wcas pushBack [WCA_ADVISORY, "FCR FAULT", "FCR FAULT"];
 };
+if (isTouchingGround _heli) then {
```

Comment:
I still don't know how I didn't realise this command existed ... good job

Reply:
Right? Randomly happened across it